charts/cmc-claim-store/values.preview.template.yaml

```diff
@@ -15,6 +15,7 @@ postgresql:
   initdbScripts:
     init.sql: |-
       CREATE DATABASE cmc WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "claimstore" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
```

Comment:
we don't need `claimstore` database any more to run claim-store-api.

Reply:
Removing it has caused lot of disruptions on product chart level especially for claim-store pod. I prefer to reinstate it there for the time being.